src/api/java/eiteam/esteemedinnovation/api/book/BookRecipeRegistry.java

```diff
@@ -11,12 +12,12 @@
     public static HashMap<String, IRecipe> recipes = new HashMap<>();
 
     public static void addRecipe(String key, IRecipe recipe) {
-        GameRegistry.addRecipe(recipe);
         recipes.put(key, recipe);
     }
 
     public static void addRecipe(String key, ItemStack output, Object... params) {
-        ShapedOreRecipe recipe = new ShapedOreRecipe(output, params);
+        ResourceLocation group = new ResourceLocation(Constants.EI_MODID, key);
```

Comment:
Because this is in the API, it is for use beyond just EI, so this method should probably be `@Deprecated` and then an overload should be made which takes a `ResourceLocation` instead of just the `String` key.

```java
/**
 * @deprecated Use {@link #addRecipe(ResourceLocation, ItemStack, Object...} instead
 */
@Deprecated
public static void addRecipe(String key, ItemStack output, Object... params) {
    addRecipe(new ResourceLocation(Constants.EI_MODID, key), output, params);
}

public static void addRecipe(ResourceLocation group, ItemStack output, Object... params) {
    ShapedOreRecipe recipe = new ShapedOreRecipe(group, output, params);
    addRecipe(group.getName()/* I can't remember the exact name of the function right now */, recipe);
}
```

If we want a non-deprecated helper method that defaults to EI_MODID we can add one in the mod itself rather than the API.